src/pseudo/Engine.java

```diff
@@ -6,7 +6,7 @@
 public class Engine {
 	//const values
 	
-	public static final byte ZERO = 0;
+	public static final byte ZERO = 0;   // <-- do we really need to make constants for that?
```

Comment:
I'd leave that just for the sake of uniformity (so that every flag is set to a const value)